dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -1768,4 +1768,80 @@ private void runPatchMetadataTests(EPerson asUser, int expectedStatus) throws Ex
         new MetadataPatchSuite().runWith(getClient(token), "/api/core/items/" + item.getID(), expectedStatus);
     }
 
+    @Test
+    public void testCreateItemInArchiveFalseBadRequestException() throws Exception {
```

Comment:
it is not immediate to understand why this test expect a bad request, it will be nice to add a javadoc for the test or a comment before the itemRest.setInArchive(false); to highlight that this is the issue (we cannot create a not final item on this endpoint)